locales/de_DE.json

```diff
@@ -3455,13 +3455,13 @@
           "label": "Name",
           "tier0-summary": "Um deinen Namen zu aktualisieren, beantrage bitte <0>Verifizierungsstufe 1</0>.",
           "tier0-details": "Wenn dein Antrag für Stufe 1 genehmigt wird, aktualisieren wir deinen Namen entsprechend den Dokumenten, die du uns übermittelt hast.",
-          "tier1-summary": "Wenn sich dein Name geändert hat und ihn aktualisieren möchtest, <0>kontaktiere bitte unseren Kundenservice</0>.",
+          "tier1-summary": "Wenn sich Ihr Name geändert hat und Sie ihn aktualisieren möchten, <0>kontaktieren Sie bitte unseren Kundenservice</0>.",
           "tier1-details": "Wir setzen deine Verifizierungsstufe zurück und bitten dich, den Verifizierungsvorgang mit deinen neuen Dokumenten zu wiederholen. Wenn dein Antrag genehmigt wird, aktualisieren wir deinen Namen entsprechend den von dir bereitgestellten Dokumenten."
         },
         "update-country": {
           "title": "Land ändern",
           "label": "Land",
-          "tier0": "Wenn sich dein Wohnsitz geändert hat und ihn aktualisieren möchtest, <0>kontaktiere bitte unseren Kundenservice</0>.",
+          "tier0": "Wenn sich Ihr Wohnsitz geändert hat und Sie ihn aktualisieren möchten, <0>kontaktieren Sie bitte unseren Kundenservice</0>.",
```

Comment:
@mucke12 This PR is reverting things to the old form! Can you rebase and try again with just the lines in the 3700-4000 range?